plugins/onepad/onepad.cpp

```diff
@@ -246,9 +246,15 @@ EXPORT_C_(s32) PADinit(u32 flags)
 
 EXPORT_C_(void) PADshutdown()
 {
-    CloseLogging();
-	if (conf) delete conf;
-	if (key_status) delete key_status;
+	CloseLogging();
+	if (conf) {
```

Comment:
Are the if condition useful? Normally you're allowed to delete a null ptr. (it is a nop)